### PR TITLE
Update Localisation Guidelines

### PR DESCRIPTION
**Pull request by BatoolMM**
### Summary

This PR adds pointers to the localisation guidelines in the README and welcome page, along with updates to certain instructions in the guide. 

Fixes #2961 

### List of changes proposed in this PR (pull-request)

- [x] Added links to the localization guidelines in the README
- [x] Added links to the localization guidelines in the welcome page
- [x] Relocated the localization chapter 
- [x] Updated instructions for our localization co-working meetings

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.

### CODE REVIEW

**Comment by BatoolMM** [comment]
The CI is failing because of a problem with reference `datastudygrouproche2020 ` which is getting fixed in #3150

File: book/website/community-handbook/translation/translation-workflow.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 We are members of The Turing Way community with different motivations to localise the content to different languages.
 Feel free to join the `#translation` channel in [_The Turing Way_ slack](https://theturingway.slack.com).
 You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-getting-started).
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:6]:
```suggestion
You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-localisation-journey).
```
Looks like this chapter was renamed?

**Comment by da5nsy** [comment]
This should now pass CI once the chapter title is updated - https://github.com/alan-turing-institute/the-turing-way/pull/3153#discussion_r1207063348 😄

File: book/website/community-handbook/translation/translation-workflow.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 We are members of The Turing Way community with different motivations to localise the content to different languages.
 Feel free to join the `#translation` channel in [_The Turing Way_ slack](https://theturingway.slack.com).
-You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-getting-started).
-The team has a bi-weekly call on Tuesdays at 5pm UTC to co-work, discuss, and check the progress of the translation.
+You will also have to create a Crowdin account but more on that [in the next section](#ch-translation-localisation-journey).
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:6]:
```suggestion
You will also have to create a Crowdin account but more on that {ref}`in the next section<ch-translation-localisation-journey>`
```

File: book/website/community-handbook/translation/translation-workflow.md [hunk 3]
```diff
@@ -13,8 +13,7 @@ The deployment of these translated versions is work in progress, and some parts
 
 ## Asynchronous translation
 
-Everyone interested in participating in a translation effort should be able to work asynchronously, so don't worry if you cannot join the bi-weekly calls.
-
+Everyone interested in participating in a translation effort should be able to work asynchronously, so don't worry if you cannot join the bi-weekly calls. More info can be found in the next chapter about [our localisation workflow](https://deploy-preview-2202--the-turing-way.netlify.app/community-handbook/translation/getting-started.html).
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:16]:
```suggestion
Everyone interested in participating in a translation effort should be able to work asynchronously, so don't worry if you cannot join the fortnightly calls. More info can be found in the next chapter about {ref}`our localisation workflow<ch-translation-localisation-journey>`.
```

File: book/website/community-handbook/translation/translation-workflow.md [hunk 4]
```diff
@@ -40,8 +39,7 @@ Document the translation guidelines for your selected language to be discussed a
 Every team has a repository with these guidelines in the GitHub organisation. 
 Check examples for
 [Portuguese](https://github.com/TWTranslation/Portuguese_specific_translation_guidelines), [Arabic](https://github.com/TWTranslation/Arabic-specific-Translation-rules), and [Turkish](https://github.com/TWTranslation/Turkish-specific-Translation-rules).
-We have also set up a public page for _The Turing Way_ in [Crowdin](https://turingway.crowdin.com/turing-way) with a README file summarising some of the translation guidelines
-Make sure you read these guidelines before you start translating for the first time.
+We have also set up a public page for _The Turing Way_ in [Crowdin](https://turingway.crowdin.com/turing-way) with a README file summarising some of the translation guidelines. Make sure you read these guidelines before you start translating for the first time.
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:42]:
```suggestion
We have also set up a public page for _The Turing Way_ in [Crowdin](https://turingway.crowdin.com/turing-way) with a README file summarising some of the translation guidelines. 
Make sure you read these guidelines before you start translating for the first time.
```
https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks

File: book/website/community-handbook/translation/translation-workflow.md [hunk 5]
```diff
@@ -93,7 +91,7 @@ DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807)
 
 
 In each co-working or external session, prioritise those chapters that are outdated or close to completion. 
-Check out our [guidelines](ch-translation-getting-started) for a list of the priority list.
+Check out our [guidelines](ch-translation-localisation-journey) for a list of the priority list.
```
Comment (da5nsy) [inline book/website/community-handbook/translation/translation-workflow.md:94]:
```suggestion
Check out our {ref}`guidelines<ch-translation-localisation-journey>` for a list of the priority list.
```

File: book/website/index.md [hunk 6]
```diff
@@ -62,6 +63,11 @@ We have contributions from across the UK, and from India, Mexico, Australia, USA
 Chapters have been written, reviewed and curated by members of research institutes and universities, government departments, and industry.
 We are committed to creating a space where people with diverse expertise and lived experiences can share their knowledge with others to allow us all to use data science to improve the world.
 
+```{admonition} Translation
+:class: tip
+Call for translators! We’re looking for translators to help translate this space for everyone! Read more about our translation workflow and how you can contribute to it [here](https://the-turing-way.netlify.app/community-handbook/translation.html).
```
Comment (da5nsy) [inline book/website/index.md:68]:
```suggestion
Call for translators! We’re looking for translators to help translate this space for everyone! Read more about our translation workflow and how you can contribute to it {ref}`here<ch-translation>`.
```

File: book/website/index.md [hunk 7]
```diff
@@ -17,6 +17,7 @@ Browse the different guides that make up the book, or use the search box to sear
 All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use _The Turing Way_ to understand their roles and responsibility of reproducibility in data science.
 You can inspect our resources on [GitHub](https://github.com/alan-turing-institute/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/alan-turing-institute/the-turing-way/blob/main/LICENSE.md)).
 
+
```
Comment (da5nsy) [inline book/website/index.md:20]:
```suggestion
```

**Review by da5nsy — CHANGES_REQUESTED** [review]

**Review by da5nsy — APPROVED** [review]

**Comment by AndreaSanchezTapia** [comment]
Hi @da5nsy I'll review it and let you know when it's done. Thanks!

**Comment by BatoolMM** [comment]
Thank you @da5nsy and @AndreaSanchezTapia!

**Review by melibleq — APPROVED** [review]
I only made a couple suggestions for the content in [book/website/index.md](https://github.com/alan-turing-institute/the-turing-way/pull/3153/files#diff-669d774a1fe68422fb6a4a569ff95b7d51a8e8d358dcbcc5c4d1443d75a2e8cd). Checked links in the deployed version to see if they're pointing to the correct pages and it all looks good to me!

File: book/website/index.md [hunk 8]
```diff
@@ -62,6 +62,11 @@ We have contributions from across the UK, and from India, Mexico, Australia, USA
 Chapters have been written, reviewed and curated by members of research institutes and universities, government departments, and industry.
 We are committed to creating a space where people with diverse expertise and lived experiences can share their knowledge with others to allow us all to use data science to improve the world.
 
+```{admonition} Translation
+:class: tip
+Call for translators! We’re looking for translators to help translate this space for everyone! Read more about our translation workflow and how you can contribute to it {ref}`here<ch-translation>`.
```
[the remark this answers is not in the document]
Comment (melibleq) [reply book/website/index.md:67]: The second substitution (from "here" to in the "translation chapter") was to follow accessibility guidelines. I think it's more recommended to actually say what the link is about than to call it "here".

**Comment by BatoolMM** [comment]
Thank you @melibleq @AndreaSanchezTapia and @da5nsy for the review 🌷 
@da5nsy would you mind rebasing the PR if needed and merging it, when you have time, pls?

**Pull request merged**